### ISSUE TITLE
Master fails to build

### ISSUE THREAD

**Issue by virtualritz** [olson-sean-k/plexus#28]
```
error[E0432]: unresolved import `plexus::encoding::ply`
  --> examples/viewer/main.rs:14:23
   |
14 | use plexus::encoding::ply::PointEncoding;
   |                       ^^^ could not find `ply` in `encoding`

warning: unused import: `plexus::prelude::*`
  --> examples/viewer/main.rs:16:5
   |
16 | use plexus::prelude::*;
   |     ^^^^^^^^^^^^^^^^^^
   |
   = note: #[warn(unused_imports)] on by default

error[E0599]: no function or associated item named `from_ply` found for type `plexus::graph::MeshGraph<Geometry>` in the current scope
  --> examples/viewer/main.rs:66:36
   |
66 |             MeshGraph::<Geometry>::from_ply(PointEncoding::<Point3<f32>>::default(), ply)
   |                                    ^^^^^^^^ function or associated item not found in `plexus::graph::MeshGraph<Geometry>`

error: aborting due to 2 previous errors
```

**Comment by olson-sean-k** [comment]
I'm guessing this is the output from `cargo test`. Cargo does [not support specifying feature flags required for examples](https://github.com/rust-lang/cargo/issues/4663) and the `viewer` example now uses the `encoding-ply` feature. To work around this, use `cargo test --all-features` instead (and consider enabling `encoding-ply` in the dependency specification for Plexus). Please let me know if `cargo build` is failing for you; AFAICT, it is currently succeeding on `master`.

This is what the recent [`viewer`](https://github.com/olson-sean-k/plexus/tree/viewer) branch is attempting to fix: it moves the code from the `viewer` example into a package so that it can be shared by examples for rendering and take a dependency on Plexus with the necessary features. Cargo uses a similar approach itself.

**Comment by olson-sean-k** [comment]
This should be fixed by 9d898a4.
